src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.java

```diff
@@ -111,6 +116,10 @@ public JPanel getPanel() {
         return panel;
     }
 
+    public JScrollPane getScrollPane() {
+        return scrollPane;
```

Comment:
The current code only contains getScrollPane(), as I removed getPanel(). I decided against returning a scrollpane in getPanel, because it makes more sense to name the method getScrollPane imho. Is this okay with you?